### ISSUE TITLE
ResizeOptions not working with Crop/Focal Point for Media Upload

### ISSUE THREAD

**Issue by FiniteVoid** [payloadcms/payload#7592]
### Link to reproduction

[Repro](https://github.com/SuyashVoid/payload-crop-issue-repro) (or just add the provided config to a new blank/website template project).

### Payload Version

3.0.0-beta.67 (repro), 3.0.0-beta.77 (personal)

### Node Version

20.12.0 (LTS)

### Next.js Version

15.0.0-canary58 (repro), 15.0.0-canary104 (personal)

### Describe the Bug

When Crop/focal point is changed, the resizeOptions, formatOptions are mostly ignored. 
That is to say, with these options:
```js
resizeOptions: {
      width: 500,
      height: 500,
      fit: 'cover',
      position: 'center',
    },
    formatOptions: {
      format: 'webp',
      options: {
        quality: 70,
      },
    },
```
This happens:
1. Edit Options
![image](https://github.com/user-attachments/assets/f4801d75-0ac5-4fc9-a033-2c2145a89dc6)

2. What's stored
![image](https://github.com/user-attachments/assets/3b399b5e-53fd-49a0-9ab1-9d3f056ea77d)

### Expected:
- Uploaded image's primary version (not a specific size defined in ImageSizes) should be resized to defined dimensions (500X500) and quality (70%).
- Resize happens at a point so as to ensure that everything the user selected in editor is available in final result (which is then scaled to fit my dimension needs depending on the 'fit' property).

### Actual:
- The dimensions of final image are free-form (except for any specific ones set in ImageSizes). 
- The image doesn't get reduced in quality and sports massive sizes (as seen in attached example).

This can cause unexpected size bloat for image (and extra cost/worse performance coz of bandwidth for each uncompressed image)

### Reproduction Steps

(I used website template for repro, but it should be same for blank)

1. Create a new Payload Project using:
`npx create-payload-app@beta -n cropIssueRepro -t website --use-pnpm`
2. Add following upload config to Media Collection:
```js
upload: {
    staticDir: path.resolve(dirname, '../../public/profilePictures'),
    crop: true,
    focalPoint: true,
    adminThumbnail: 'thumbnail',
    mimeTypes: ['image/*'],
    imageSizes: [
      {
        width: 100,
        height: 100,
        name: 'thumbnail',
        formatOptions: {
          format: 'webp',
          options: {
            quality: 70,
          },
        },
      },
    ],
    resizeOptions: {
      width: 500,
      height: 500,
      fit: 'cover',
      position: 'center',
    },
    formatOptions: {
      format: 'webp',
      options: {
        quality: 70,
      },
    },
  },
```
3. Create a new item in Media Collection:
    - Select Image to upload (it should be bigger than 500X500 (and possibly a different aspect ratio for more clear demonstration of bug)).
    - Edit image and crop it to something that vaguely resembles a square (or anything else for that matter).
    - Apply Changes.
    - Save.
4. Now, newly created image will not conform to the given 500X500 resize restriction.
    - (Also, the admin thumbnail would be weirdly cropped instead of resized and not be a proper representation of the original image).

### Adapters and Plugins

db-mongodb

**Comment by JarrodMFlesch** [comment]
@PatrikKozak here is another good upload issue.

**Comment by PatrikKozak** [comment]
Fixed with this #8528.
